technic/machines/HV/quarry.lua

```diff
@@ -56,7 +56,7 @@ end
 
 local function reset_quarry(pos)
 	local meta = minetest.get_meta(pos)
-	local node = technic.get_or_load_node(pos) or minetest.get_node(pos)
+	local node = minetest.get_node(pos)
```

Comment:
Should this keep using `technic.get_or_load_node(pos)` or is `minetest.get_node(pos)` guaranteed to be safe (guaranteed that node is already loaded)?

Reply:
It is safe, it shouldn't ever be called on an unloaded node, and even if that happens `minetest.get_node` always returns a value.